diffkemp/simpll/DebugInfo.h

```diff
@@ -65,7 +67,7 @@ class DebugInfo {
 
     /// Maps local variable names to their values.
     std::unordered_map<std::string, const Value *> LocalVariableMapL,
-                                                   LocalVariableMapR;
+            LocalVariableMapR;
```

Comment:
This is not very nice since this alignment may lead to missing the fact that there are 2 variables defined here. It would suffice to put them at the same line, but I guess that `clang-format` won't allow that either without breaking other stuff.

Reply:
I agree, however, `clang-format` only supports indentation after an open `(` bracket and for semicolon-separated declarations and assignments. The reason for the lack of support here is, at least based on information I have found, to discourage the use of comma-separated variable declarations altogether. Therefore, I will change the declaration to semicolon-separated one in such cases for better readability.

Reply:
Good, this should be the recommended approach from now in case of multiline variable definitions.